### ISSUE TITLE
Ignore non-config flags

### ISSUE THREAD

**Issue by MaKleSoft** [gravitational/configure#18]
Currently if I use `configure.ParseCommandLine(config, os.Args[1:])` and run the program with a flag not declared in my config struct I get the error `unknown long flag '--my-flag'`. This obviously causes problems in case I want to support other flags apart from the ones declared in my config struct or even just use two separate structs. Therefore, I think it would be better if undeclared flags would simply be ignored.

**Comment by llorllale** [comment]
> use two separate structs

I agree. I'm forced to break encapsulation between two unrelated packages due to this.
